rootfs/etc/profile.d/aliases.sh

```diff
@@ -4,9 +4,36 @@ alias default='kubectl --namespace=default'
 alias telnet='busybox-extras telnet'
 alias ll='ls -l'
 
+# taboff turns off tab auto-completion, which is practically required when pasting code with tabs into the terminal
+alias taboff='bind '\''set disable-completion on'\'''
+# tabon turn on tab auto-completion
+alias tabon='bind '\''set disable-completion off'\'''
+
+
+# Run chamber to put secrets for given service(s) in the environment
+function chdo() {
```

Comment:
How about adding a small snippet in `docs/chdo.md` that basically explains this. I know it's tedious, but I think it will get easier once we have a few.

Reply:
Atleast it would help in discovery of commands and provide feedback on how we can make our `man` pages better